doc/user-guide/Push-notifications.md

```diff
@@ -0,0 +1,188 @@
+## Push Notifications
```

Comment:
If you want this file to be visible on readthedocs, you need to add it to mkdocs.yml file.